lib/home/profile/info/heart_center_info.dart

```diff
@@ -34,219 +30,169 @@ class _HeartCenterInfoState extends State<HeartCenterInfo> {
                 ),
-              ),
-            ),
-            const Padding(
-              padding: EdgeInsets.all(16.0),
-              child: Text(
-                'Email: info@theheartcenter.one\nLocation: 1701 South Figueroa #1437 Los Angeles, CA 90015',
-                textAlign: TextAlign.center,
-              ),
-            ),
-            GestureDetector(
-              onTap: () => _launchURL('https://theheartcenter.one/'),
-              child: const Text(
-                'Visit Our Website',
-                style: TextStyle(
-                  fontSize: 18,
-                  fontWeight: FontWeight.bold,
-                  color: Colors.blue,
-                  decoration: TextDecoration.underline,
+                const SizedBox(height: 16),
+                TextFormField(
+                  controller: _messageController,
+                  decoration: const InputDecoration(labelText: 'Your Message'),
+                  maxLines: 4,
+                  validator: (value) {
+                    if (value == null || value.isEmpty) {
+                      return 'Please enter a message';
+                    }
+                    return null;
+                  },
                 ),
-              )
-            ),
-            const SizedBox(height: 16.0),
-            ElevatedButton(
-              onPressed: () => _launchURL('https://secure.givelively.org/donate/heart-center-inc'),
-              child: const Text('Donate'),
+                const SizedBox(height: 16.0),
+                ElevatedButton(
+                  onPressed: () {
+                    if (_formKey.currentState!.validate()) {
+                      _submitForm(_emailController.text, _messageController.text);
+
+                      _emailController.clear();
+                      _messageController.clear();
+                    }
+                  },
+                  child: const Text('Submit'),
+                ),
+              ],
             ),
-            const SizedBox(height: 8.0),
-            ElevatedButton(
-              onPressed: () => _launchURL('https://docs.google.com/forms/d/e/1FAIpQLScOEwS0sew6teRQoov6h1kzLt0xOAw0hk-bmKM14FZ28YAqDA/viewform'),
-              child: const Text('Volunteer'),
+          ),
+          const SizedBox(height: 16),
+          const SelectableText(
+            'Email: info@theheartcenter.one\n'
+            'Location: 1701 South Figueroa #1437 Los Angeles, CA 90015',
+            textAlign: TextAlign.center,
+          ),
+          TextButton(
+            style: TextButton.styleFrom(foregroundColor: Colors.blue),
+            onPressed: () => launchUrlString('https://theheartcenter.one/'),
+            child: const Text(
+              'Visit Our Website',
+              style: StyleText(
+                size: 16,
+                weight: 500,
+                decoration: TextDecoration.underline,
+                decorationColor: Colors.blue,
+              ),
             ),
```

Comment:
In general, I like using the button widgets rather than `GestureDetector`s, since there are some nice subtle effects when you tap/hover your mouse cursor on them (and they come with a built-in `FocusNode` for accessibility).